systers_portal/templates/meetup/base.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
+{% load verbose_name %}
+
+{% block title %}
+  - {{ object.name }}
+{% endblock %}
+
+{% block content %}
+  <div class="mt40"></div>
+  <div class="row">
+    <div class="col-md-12">
+      <h1>{{ object.name }}</h1>
+      <hr/>
+    </div>
+    <div class="col-md-9">
+      <ul class="nav nav-pills">
+        <li> <a href="#">Upcoming</a> </li>
+        <li> <a href="#">Past</a></li>
+        <li> <a href="{% url 'display_meetup_location' slug 'organizers' %}">Organizers</a></li>
```

Comment:
Sorry but I am not sure for which url you are talking about.

If it is for line 19, than it is correct url - it will be meetup/<slug value>/organizers which will become meetup/boston/organizers is slug value is boston. Probably confusion arose due the values of two slugs hold. I have explained that in above comments.